### PR TITLE
Update multi-geo-add-group-with-pdl.md

### PR DESCRIPTION
**Pull request by msbemba**
Added link to create group with specific PDL

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7034

File: microsoft-365/enterprise/multi-geo-add-group-with-pdl.md [hunk 1]
```diff
@@ -53,3 +53,4 @@ Note that SharePoint group site provisioning is on-demand. The site will be prov
 ## Related topics
 
 [Connect to Exchange Online PowerShell](/powershell/exchange/connect-to-exchange-online-powershell)
+[Create groups with a specific preferred data location using Grpah API](/graph/api/group-post-groups)
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/multi-geo-add-group-with-pdl.md]:
```suggestion

[Create groups with a specific preferred data location using Graph API](/graph/api/group-post-groups)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Pull request merged**